apps/help-with-fees/identity/aat.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "aadpodidentity.k8s.io/v1"
```

Comment:
All envs need the per env patch added, see:
https://github.com/hmcts/cnp-flux-config/blob/3edbc3f79ea0c839b970a487147ffbeae66bf3ea/apps/rpe/aat/base/kustomization.yaml#L8

Reply:
Added